java/java.lsp.server/nbproject/project.xml

```diff
@@ -454,6 +454,17 @@
                         <specification-version>1.101</specification-version>
                     </run-dependency>
                 </dependency>
+
+                <dependency>
+                    <code-name-base>org.netbeans.modules.projectuiapi</code-name-base>
+                    <build-prerequisite/>
+                    <compile-dependency/>
+                    <run-dependency>
+                        <release-version>1</release-version>
+                        <implementation-version/>
```

Comment:
Implementation version? It'd be better to use real API, not implementation dependencies.

Reply:
This will be tough. Event `projectui` impl-depends on `projectuiapi` for this type. Should we make `BrokenReferencesImplementation` a regular API ?

Reply:
Making `BrokenReferencesImplementation` (a two method interface) an SPI in `org.netbeans.spi.project.ui` is certainly better than to spread implementation dependencies.

Reply:
SPI created in 27ad3df6